src/util/useAuth.ts

```diff
@@ -115,10 +120,11 @@ export const useAuth = (): [AuthState, boolean] => {
           refetchUser()
         } else {
           setAuthState(AuthState.UNAUTHENTICATED)
+          setIsLoginFailed(true)
```

Comment:
```suggestion
          setErrorMessage(text('login_failed'))
```
Can't the message be set here? A new state variable for this seems excessive.

Reply:
Hmm true! This code changed a bit on the way and didn't think setting it here after setIsLoginFailed variable was made!